tools/crm_mon.c

```diff
@@ -127,6 +127,57 @@ static void mon_st_callback_event(stonith_t * st, stonith_event_t * e);
 static void mon_st_callback_display(stonith_t * st, stonith_event_t * e);
 static void kick_refresh(gboolean data_updated);
 
+static gboolean
+include_cb(const gchar *option_name, const gchar *optarg, gpointer data, GError **error) {
+    char **parts = NULL;
+
+    parts = g_strsplit(optarg, ",", 0);
+    for (char **s = parts; *s != NULL; s++) {
+        if (crm_str_eq(*s, "all", TRUE)) {
+            show = mon_show_all;
+        } else if (crm_str_eq(*s, "attributes", TRUE)) {
+            show |= mon_show_attributes;
+        } else if (crm_str_eq(*s, "bans", TRUE)) {
+            show |= mon_show_bans;
+        } else if (crm_str_eq(*s, "counts", TRUE)) {
+            show |= mon_show_counts;
+        } else if (crm_str_eq(*s, "dc", TRUE)) {
+            show |= mon_show_dc;
+        } else if (crm_str_eq(*s, "default", TRUE)) {
```

Comment:
Let's allow "default" or "defaults" to match (we can still just show "default" in the help)

Reply:
I was also thinking about adding "none" - as in --include=none or --exclude=none.  I don't know that it gets us anything, but it seemed like a nice parallel to "all".

Reply:
It does have a symmetry to it. --include=none would be equivalent to --exclude=all and vice versa, which might be a little confusing, but it would let people use whichever is natural to their mind